src/mpyl/project.py

```diff
@@ -640,30 +639,29 @@ def load_possible_parent(
 
 
 def load_project(
-    root_dir: Path,
     project_path: Path,
-    strict: bool = True,
+    strict: bool = False,
```

Comment:
don't we want it to be strict by default?

Reply:
I would say so, but currently we only do `strict=True` when we call `mpyl run projects lint`, everywhere else we skip schema validation. That's why I inverted the default, to signal that in most places validation is actually disabled.

Reply:
Done c14715419dc1e283ff2283de76ee97e0e4b43226.